x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -217,6 +249,29 @@ export class ExecuteReportTask implements ReportingTask {
     return await store.setReportFailed(report, doc);
   }
 
+  private async _saveExecutionError(
+    report: SavedReport,
+    failedToExecuteErr: any
+  ): Promise<UpdateResponse<ReportDocument>> {
+    const message = `Saving execution error for ${report.jobtype} job ${report._id}`;
+    const errorParsed = parseError(failedToExecuteErr);
+    // log the error
+    errorLogger(this.logger, message, failedToExecuteErr);
+
+    // update the report in the store
+    const store = await this.getStore();
+    const completedTime = moment().toISOString();
+    const doc: ReportFailedFields = {
+      completed_at: completedTime,
```

Comment:
`completed_at` should not be set at this step

Reply:
Done in https://github.com/elastic/kibana/pull/173775/commits/7703a77e1c1e201f9f641d45c078f8ab0a205cf6